### DISCUSSION TITLE
Functional Component Documentation

### DISCUSSION THREAD

**Discussion by dev-shetty** [gilbarbara/react-joyride#893] [Q&A] [answered]
Where can i Find the documentation especially for React Functional Components

**Comment by gilbarbara** [comment, marked as answer]
Hey @Deveesh-Shetty 
There's no dedicated documentation for FC

**Reply by dev-shetty** [reply]
Yea, I found the docs they are similar for both functional and class components...

I figured out everything, but 
- How do I stop the rendering of modals after the user has completed the last step, and also the same rendering again when navigating back to the page from another page. 
Is there any function available to track that

Thank you

**Comment by gilbarbara** [comment]
@Deveesh-Shetty 

The library isn't suppose to render tooltips after it finishes iterating through the steps. Are you using `stepIndex`?
Anyway, you can listen to events in the callback and set a persistent flag in localStorage or a database to indicate the user has already completed the tour.

**Reply by dev-shetty** [reply]
No i am not using `step index`
In my app, I am using continuous modal as soon as the the user enters in (showing some key-features about app)....

I will check for the events in callbacks.... Thank you for helping
